README.md

```diff
@@ -107,3 +107,14 @@ Many thanks to the macOS community and special thanks to [mczachurski](https://g
 
 ## License
 [MIT](LICENSE)
+
+## Translation
+
+Equinox is translated to `English` & `French`. To translate Equinox to another language:
```

Comment:
also, you could put yourself as the author of `French` translation.

Reply:
Of course 👍 wasn't sure how much ownership to take 😄